blog.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Blog</title>
+    <link rel="stylesheet" href="./styles/master.css">
+  </head>
+
+  <body>
+    <header>
+      <img src="images/adoptadog-logo.svg" width="200" alt="adoptadog logo">
+      <p>Seattle's premiere dog adoption agency</p>
+
+      <img src="images/cart-icon.svg" width="24" alt="shopping cart icon">
+      <p>$1234.56</p>
+      <button type="button">Checkout</button>
+
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+
+
+    <main>
+      <h2>Adoptadog Blog</h2>
+        <section class="blog">
+          <img src="images/blog-1.jpg" width="300" alt="a woman holding a dog facing the grand canyon">
+          <h2>Traveling With Your Dog</h2>
```

Comment:
Would suggest wrapping your h2 and p tags in another div. This will allow you to push the text onto the side of your images